tests/install/test_upgrade.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import pytest
+
+from pytest_bdd import parsers
+from pytest_bdd import scenario
+from pytest_bdd import when
+
+from utils.helper import create_version_archive
+from utils.helper import run_ansible_playbook
+
+
+@pytest.fixture
+def pytestbdd_strict_gherkin():
+    return False
+
+
+@scenario('features/upgrade.feature', 'Upgrade from 1.0.0')
+def test_upgrade(inventory_tagging):
+    pass
+
+
+@when(parsers.parse(
+    "I launch ansible playbook '{playbook}' from version {version}"
+))
+def ansible_playbook_from_version(
+        request, inventory, playbook, archive_dir, version):
+    archive_version = create_version_archive(version, archive_dir)
+    inventory_path = os.path.abspath(inventory)
```

Comment:
This is not used? However, shouldn't it be properly exported in the env?

Reply:
This was the other way around for fixing the CI: converting the ANSIBLE_INVENTORY variable into an absolute path when needed

Reply:
I kept the version of ANSIBLE_INVENTORY exported as an absolute path

Reply:
I prefer the absolute path based approach. Remember how there was a 'demo effect' hickup during the SF demo of MetalK8s many months ago, because I used a relative path for `$KUBECONFIG` and moved into another directory? :wink: